### DISCUSSION TITLE
Assessment of the difficulty in porting CPU architecture for Trilinos

### DISCUSSION THREAD

**Discussion by wangyuliu** [trilinos/Trilinos#12228] [General]
Hello everyone! I am working on implementing a tool to assess the complexity of CPU architecture porting. It primarily focuses on RISC-V architecture porting.In fact, it can be estimated on average across various architectures. As part of my dataset, I have collected the Trilinos project. I would like to gather community opinions to support my assessment. I appreciate your help and response! Based on scanning tools, the porting complexity is determined to be moderate, with a moderate amount of code related to the CPU architecture in the project. It would require a professional person or team to handle this task (referring to the overall workload from adapting the project to a specific architecture to achieving full functionality on that architecture). Is this assessment accurate? I look forward to your help and response.

**Comment by cgcgcg** [comment]
@lucbv
